adapters/smartadserver/smartadserver_test.go

```diff
@@ -18,3 +20,27 @@ func TestJsonSamples(t *testing.T) {
 
 	adapterstest.RunJSONBidderTest(t, "smartadservertest", bidder)
 }
+
+func TestGetBidTypeFromMarkupType_WhenBanner_ShouldReturnBanner(t *testing.T) {
+	mediaType := getBidTypeFromMarkupType(openrtb2.MarkupBanner)
+
+	assert.Equal(t, openrtb_ext.BidTypeBanner, mediaType)
+}
+
+func TestGetBidTypeFromMarkupType_WhenVideo_ShouldReturnVideo(t *testing.T) {
+	mediaType := getBidTypeFromMarkupType(openrtb2.MarkupVideo)
+
+	assert.Equal(t, openrtb_ext.BidTypeVideo, mediaType)
+}
+
+func TestGetBidTypeFromMarkupType_WhenAudio_ShouldReturnAudio(t *testing.T) {
+	mediaType := getBidTypeFromMarkupType(openrtb2.MarkupAudio)
+
+	assert.Equal(t, openrtb_ext.BidTypeAudio, mediaType)
+}
+
+func TestGetBidTypeFromMarkupType_WhenNative_ShouldReturnNative(t *testing.T) {
+	mediaType := getBidTypeFromMarkupType(openrtb2.MarkupNative)
+
+	assert.Equal(t, openrtb_ext.BidTypeNative, mediaType)
+}
```

Comment:
Why did You add those test? 
correctly is to test this through jsons

Reply:
I removed it. It's indeed already tested by jsons